app/behaviors/base.ts

```diff
@@ -5,18 +5,34 @@ export function timeToFrame(time: number): number {
 
 export default interface Behavior {
   /**
+   * Calculates the frames for the given parameters.
    *
    * @param from
    * @param to
    * @param duration Duration in milliseconds
    * @param velocity
    * @param delay
+   * @param previousFramesFromTime The previous frames (if any) that would have happened without an interruption.
    */
   toFrames(
     from: number,
     to: number,
     duration: number,
     velocity: number,
-    delay?: number
+    delay?: number,
+    previousFramesFromTime?: number[]
```

Comment:
This method signature is getting pretty unwieldy. Perhaps it should accept a params object?

Reply:
Yeah agreed. I'm going to decide on an interface for this one on my spring branch as that will influence this too. It's likely to become an object indeed.

Reply:
I've done part of the improvements in this PR: https://github.com/cardstack/animations-experiment/pull/18